src/components/Layout/PageLayout/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, CssBaseline, Container } from "@mui/material";
+import { FC, ReactNode } from "react";
+import { Header } from "../Header";
+import { pageLayoutStyles } from "./pageLayoutStyles";
+
+interface PageLayoutProps {
+  children: ReactNode;
+}
+
+export const PageLayout: FC<PageLayoutProps> = ({ children }) => {
+  return (
+    <Box sx={pageLayoutStyles.containerWrapper}>
+      <Header />
```

Comment:
The Layout should be generic and only the layout gets the Header component from outside.